app/models/package_manager/nu_get.rb

```diff
@@ -137,15 +137,15 @@ def self.mapping(raw_project)
       nuspec_repo = raw_nuspec&.locate("package/metadata/repository")&.first
       nuspec_repo = nuspec_repo["url"] if nuspec_repo
 
-      {
+      MappingBuilder.build_hash(
         name: raw_project[:name],
         description: item.description,
         homepage: item.project_url,
         keywords_array: item.tags,
         repository_url: repo_fallback(nuspec_repo, item.project_url),
-        releases: raw_project[:releases],
```

Comment:
here's one example of a key that's not needed. It was added years ago in https://github.com/librariesio/libraries.io/commit/70396e221d59b568f2f110022260395a2dc3e64a#diff-8b25ada8935cd42afbba863d31a65269a35dbc467397160353adf3425030ed25R68 , but we actually only allow `:versions` as a key here.